packages/devtools_app/test/debugger/debugger_scripts_test.dart

```diff
@@ -44,14 +44,16 @@ void main() {
   ) async {
     await tester.pumpWidget(
       wrapWithControllers(
-        const DebuggerScreenBody(),
+        const NotificationsView(
```

Comment:
wrapWithControllers already includes a NotificationsView in the widget tree, so we shouldn't need to add this here

Reply:
done!